wiki/Beatmap/Beatmap_tags/en.md

```diff
@@ -0,0 +1,207 @@
+# Beatmap tags
+
+*Not to be confused with [TAG difficulties](/wiki/Beatmap/TAG_beatmaps).*
+
+Keywords can be added to beatmaps to make searching for them more convenient. There are two types of tags, each with their own purpose.
+
+## Mapper tags
+
+*For rules regarding mapper tags, see: [Ranking criteria/Metadata § Tags](/wiki/Ranking_criteria/Metadata#tags)*
+
+**Mapper tags** can be used to include useful background information about a beatmap or song, e.g. the album name or the medium for which the song was produced for.
```

Comment:
```suggestion
**Mapper tags** are tags that describe useful background information about a beatmap or a song, e.g. the name of the album or the medium for which the song was produced for. As the name suggests, these tags are added by the beatmap host during the [ranking process](/wiki/Beatmap_ranking_procedure).
```

Reply:
Good idea mentioning the ranking process!